aten/src/ATen/core/boxing/impl/make_boxed_from_unboxed_functor.h

```diff
@@ -377,6 +377,10 @@ namespace impl {
       assert_is_valid_output_type<T, AllowDeprecatedTypes>();
       return c10::ivalue::from(std::move(v));
     }
+    static IValue copy(const T& v) {
+      assert_is_valid_output_type<T, AllowDeprecatedTypes>();
+      return IValue(v);
```

Comment:
I suppose we could use `c10::ivalue::from(v);`, and it will likely not actually move `v` due to `const` (but can't prevent a ctor that matches `(const value&&)`). Explicitly construct here seems clearer.